app/css/header.css

```diff
@@ -0,0 +1,41 @@
+/* .header {
+  height: 500px;
+} */
+
+.arduino-logo {
+    width: 460%;
```

Comment:
all this % looks to Hight , but apparently is working ... maybe will cause troubles but ok.